### ISSUE TITLE
Cloud, Dashboard - Import dashboard fails with associated Label and Variable

### ISSUE THREAD

**Issue by corriganjohn** [influxdata/influxdb#16560]
__Related Honeybadger Trace__
https://app.honeybadger.io/projects/61122/faults/59612486

__Steps to reproduce:__
List the minimal actions needed to reproduce the behavior.

1. Production : Version 2.0.0 (93c8a2a)
2. Create Dashboard with a Label and a Variable, export to file(created in Acceptance)
3. (In Production, where the Label and Variable don't exist) Import the Dashboard 
4. "Failed to import dashboard:TypeError: Cannot read property 'id' of undefined"
5. (Refresh screen ... Dashboard is actually there)
6. Check Settings - Label has been created, but NOT Variable
7. Try to import Dashboard again SUCCEEDS

__Expected behavior:__
Expect the Import to create the Variable and Label before creating the Dashboard (?)

__Actual behavior:__
It appears to be a timing issue on the creation of the associated objects. The expected Variable doesn't exist so the Dashboard creation fails (could this be an issue with the way the Dashboard set of objects is being exported?)

__Environment info:__

Cloud - Version 2.0.0 (93c8a2a)

__Logs:__
Include snippet of errors in log.

__Performance:__
Generate profiles with the following commands for bugs related to performance, locking, out of memory (OOM), etc.

```sh
# Commands should be run when the bug is actively happening.
# Note: This command will run for at least 30 seconds.
curl -o profiles.tar.gz "http://localhost:8086/debug/pprof/all?cpu=true"
curl -o vars.txt "http://localhost:8086/debug/vars"
iostat -xd 1 30 > iostat.txt
# Attach the `profiles.tar.gz`, `vars.txt`, and `iostat.txt` output files.
```

**Comment by corriganjohn** [comment]
I captured a screen recording of what happens when following the steps, but it's a Quicktime .mov file (which we can't upload here) ... let me know if I should try to convert it to load here or if I can just pass the recording off to someone

**Comment by corriganjohn** [comment]
[stock_market_data.json.zip](https://github.com/influxdata/influxdb/files/4071950/stock_market_data.json.zip)

**Comment by corriganjohn** [comment]
[Import_Dashboard_Fail_Production_480p.mov.zip](https://github.com/influxdata/influxdb/files/4071954/Import_Dashboard_Fail_Production_480p.mov.zip)

**Comment by drdelambre** [comment]
this was closed by #16674
